automation_tools/__init__.py

```diff
@@ -2244,6 +2251,10 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         installer_options.update({
             'foreman-proxy-plugin-pulp-puppet-content-dir': '/etc/puppetlabs/code/environments'})
 
+    # enable cockpit feature
+    if (sat_version not in ('6.3', '6.4', '6.5', '6.6')):
+        installer_options.update({'enable-foreman-plugin-remote-execution-cockpit': 'true'})
```

Comment:
@rplevka for upstream-nightly, this condition become true and caused failure ERROR: too many arguments.

It works fine if passed `--enable-foreman-plugin-remote-execution-cockpit` instead of `--enable-foreman-plugin-remote-execution-cockpit="true"`. 

I see same behaviour for satellite 6.7. Is it expected or bug?